extensions/blocks/rating-star/index.js

```diff
@@ -27,7 +28,7 @@ export const settings = {
 		_x( 'rating', 'block search term', 'jetpack' ),
 		_x( 'review', 'block search term', 'jetpack' ),
 	],
-	category: supportsCollections() ? 'formatting' : 'jetpack',
+	category: supportsCollections() ? getCategoryWithFallbacks( 'widgets', 'formatting' ) : 'jetpack',
```

Comment:
`widgets` category is a pre-existing category – it's part of both legacy and new categories, right? We don't need the fallback in that case.

```suggestion
	category: supportsCollections() ? 'widgets' : 'jetpack',
```

(I think this already came up, but my brain just processed it 🙂 )

Reply:
This is the same situation as with the [Newspack Blocks categories in FSE](https://github.com/Automattic/wp-calypso/pull/43670#pullrequestreview-444862833). Should we also [remove their fallbacks](https://github.com/Automattic/wp-calypso/pull/43670#issuecomment-655625284)?

Reply:
@sirreal Maybe I misunderstood you slightly [here](https://github.com/Automattic/wp-calypso/pull/43670#issuecomment-655625284). When I said to keep it as-is, I meant with the fallback, but now that I'm revisiting this, I see you and Bart have a good point: the fallback won't be of any use in this scenario unless we revert the order. **I guess the simplest way to go is to remove the use of fallbacks in these cases then.**

Reply:
I hadn't understood myself 🙂 

Let's go with this:

> remove the use of fallbacks in these cases then.

Reply:
[Done](https://github.com/Automattic/jetpack/pull/16363/commits/9f5648fca2cf6e530b269125d2e8b7eeb8f3641e). In Jetpack the only one that was assigned to `widgets` was the `rising-start` block.